pipes/WDL/tasks/tasks_interhost.wdl

```diff
@@ -119,10 +135,13 @@ task beast {
     bash -c "sleep 60; nvidia-smi" &
     beast \
       -beagle_multipartition off \
-      -beagle_GPU -beagle_cuda -beagle_SSE \
-      -beagle_double -beagle_scaling always \
-      -beagle_order 1,2,3,4 \
-      ${beauti_xml}
+      -beagle_GPU \
+      -beagle_cuda \
+      -beagle_SSE \
+      -beagle_double \
```

Comment:
One request -- can we parameterize with some `boolean beagle_double=true` that turns into either `-beagle_double` or `-beagle_single` on the command line? In >90% of the xmls I've run, we need double precision to converge well, but every once in a while, single-precision is enough and the speed gains are huge. The best performance/$ is actually on a T4 but only if you're running single-precision.